pipeline/inputs/windows-event-log-winevtlog.md

```diff
@@ -39,6 +40,12 @@ Here is a minimum configuration example.
 
 Note that some Windows Event Log channels \(like `Security`\) requires an admin privilege for reading. In this case, you need to run fluent-bit as an administrator.
 
+#### Query Languages for Event_Query Parameter
+
+`Event_Query` parameter can handle XML query for filtering Windows EventLog when collecting them.
+This parameter is able to handle both of [XPath](https://developer.mozilla.org/en-US/docs/Web/XPath) and XML Query queries.
```

Comment:
```suggestion
The supported query types are [XPath](https://developer.mozilla.org/en-US/docs/Web/XPath) and XML Query.
```

Link to XML Query?

Reply:
XML Query is having some of the specific syntax for querying Windows EventLog. However, I wasn't able to find the suitable page which describes the abstract of the syntax....